### ISSUE TITLE
Implement the QuickStart but got an error when  using BertForMaskedLM to predict a masked token

### ISSUE THREAD

**Issue by JJJJane** [huggingface/pytorch-transformers#1112]
## ❓ Questions & Help

I was running the BERT example following the instruction of pytorch-transformers' docs, but when Iusing BertForMaskedLM to predict a masked token, an error occured:
"INFO:pytorch_transformers.modeling_utils:Weights from pretrained model not used in BertForMaskedLM: ['cls.seq_relationship.weight', 'cls.seq_relationship.bias']"

Any idea how to fix this? Thanks!

**Comment by JJJJane** [comment]
I see, thanks!
